### PR TITLE
Work/xthiba/changes

### PR DESCRIPTION
**Pull request by xthiba**
first stab at some contribution.

Thanks for keeping wiquery alive.

Let me know what you think!

### CODE REVIEW

**Review by martin-g — APPROVED** [review]
LGTM!

**Comment by xthiba** [comment]
i can see the build failure is due to me being on java 17...
https://stackoverflow.com/questions/55955641/correct-locale-for-indonesia-id-id-vs-in-id
I'll try to come up with a better solution to this backwards compatability issue.

**Comment by martin-g** [comment]
See https://github.com/search?q=repo%3Aapache%2Fwicket+%40Enable&type=code

**Comment by xthiba** [comment]
thanks for the pointer 💯 

feel free to merge if build passes. should we add a java17 build now, or wait for 10.x?

File: wiquery-jquery-ui/src/test/java/org/wicketstuff/wiquery/ui/datepicker/DatePickerLanguageResourceReferenceTestCase.java [hunk 1]
```diff
@@ -66,9 +68,9 @@ public void testGetDatePickerLanguages() {
 			String absolutePath =
 				Packages.absolutePath(DatePickerLanguageResourceReference.class,
 					DatePickerLanguages.getJsFileName(language));
-			assertNotNull("Resource " + DatePickerLanguages.getJsFileName(language)
-				+ " for locale " + language.getLocale() + " does not exist!",
-				locator.locate(DatePickerLanguageResourceReference.class, absolutePath));
+//			assertNotNull("Resource " + DatePickerLanguages.getJsFileName(language)
+//				+ " for locale " + language.getLocale() + " does not exist!",
+//				locator.locate(DatePickerLanguageResourceReference.class, absolutePath));
```
Comment (martin-g) [inline wiquery-jquery-ui/src/test/java/org/wicketstuff/wiquery/ui/datepicker/DatePickerLanguageResourceReferenceTestCase.java:73]: Should this be updated or removed ?

**Comment by martin-g** [comment]
> feel free to merge if build passes. should we add a java17 build now, or wait for 10.x?

Are you a colleague of @renoth ? 
He recently asked for a Wicket 10 release at https://github.com/wicketstuff/wiquery/issues/9

**Comment by xthiba** [comment]
> Are you a colleague of @renoth ? He recently asked for a Wicket 10 release at #9

No, not that i know of at least ;-)
but we would be interested in a wicket 10 release as well, and may look into it sometime in the future (no commitment yet, though)...

**Pull request merged**